git-link.el

```diff
@@ -320,11 +350,10 @@ Defaults to \"origin\"."
                       (region (git-link--get-region)))
                  (list remote (car region) (cadr region))))
   (let* ((remote-host (git-link--remote-host remote))
-	 (filename    (git-link--relative-filename))
-	 (branch      (git-link--branch))
-	 (commit      (git-link--commit))
-	 (handler     (cadr (assoc remote-host git-link-remote-alist))))
-
+	 (filename (git-link--relative-filename))
+	 (branch (git-link--branch))
+	 (commit (git-link--commit))
+	 (handler (git-link--handler 1 git-link-remote-alist remote-host)))
```

Comment:
Still don't like having to pass `1` every time. We're coding for an abstraction that doesn't exist. Maybe it will, maybe it wont.